apps/osim/tests/test_models.py

```diff
@@ -17,6 +17,16 @@
 pytestmark = pytest.mark.unit
 
 
+def assert_state_equals(current, expected):
+    message = f'flaw classified as {current.name}, expected {expected["name"]}'
+    assert current.name == expected["name"], message
+
+
+def assert_workflow_equals(current, expected):
+    message = f"flaw classified in {current.name} workflow, expected {expected.name}"
+    assert current.name == expected.name, message
```

Comment:
unused in the commit that it's introduced

Reply:
changed to the correct commit